scripts/src/java/org/oppia/android/scripts/SyntaxErrorHandler.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.android.scripts
+
+import org.xml.sax.ErrorHandler
+import org.xml.sax.SAXParseException
+
+/**
+ * Class for custom error handling of the parse exception thrown
+ * by the parser, to log the failure message in a cleaner way.
+ */
```

Comment:
Here & everywhere else: KDocs should fit the full 100 character limit before being line-wrapped just like code (prefer not prematurely line-wrapping).

Reply:
Done. noted.